packages/frontend/core/src/components/affine/auth/after-sign-in-send-email.tsx

```diff
@@ -75,26 +85,26 @@ export const AfterSignInSendEmail = ({
       </div>
 
       <div className={style.authMessage} style={{ marginTop: 20 }}>
-        {/*prettier-ignore*/}
```

Comment:
Please double check if i18n still works correctly after prettier ignore comment is removed